### PR TITLE
Code merge into master

### PR DESCRIPTION
**Pull request by djuricna**
Merging "code" branch into master: contains the Code folder, with the raw data, cleaned data, paper summary data, and the two RMD files with how the raw data was cleaned and how the LMEM and plots were made

### CODE REVIEW

**Comment by evelin-d** [comment]
good to merge!

File: Code/LMEM_and_Plots.Rmd [hunk 1]
```diff
@@ -0,0 +1,372 @@
...
+                        x="Predator Feeding Interaction")+
+                        scale_x_discrete(limits=c("Planktivorous","Piscivorous","Predacious"))         
+                                           
+
+# ggsave("FeedingInteraction_LogPredMass.jpg", plot = last_plot(), 
+#device = NULL, path = NULL,
+#   scale = 1, width = NA, height = NA, units = c("in", "cm", "mm"),
+#   dpi = 300, limitsize = TRUE)
+
+
```
Comment (evelin-d) [inline Code/LMEM_and_Plots.Rmd:370]: # PLOTTING INTERACTION BETWEEN LIFE STAGE : FEEDING INTERACTION
fish_filter %>% 
  filter(Predator.lifestage != "Larva", Type.of.feeding.interaction != "planktivorous") %>% 
  group_by(Predator.lifestage, Type.of.feeding.interaction) %>% 
  summarise(meanlogpredmass = mean(logpredmass), n_obs = n(), 
            se_mass = (meanlogpredmass/sqrt(n_obs))) %>% 
  ggplot()+
  aes(x = Predator.lifestage, y = meanlogpredmass, colour = Type.of.feeding.interaction)+
  geom_line(aes(group=Type.of.feeding.interaction))+
  geom_errorbar(aes(ymin = (meanlogpredmass-se_mass), ymax = (meanlogpredmass + se_mass)),
                    width = 0.05)+
  geom_point()+
  labs(x = "Predator Lifestage", y = "Mean of Log Predator Mass", 
       colour = "Type of Feeding Interaction")+
  scale_x_discrete(limits = c("Juvenile", "Adult"))+
  scale_colour_manual(values = c("midnightblue", "orangered"))+
  theme_classic()

**Comment by NatashaSammut** [comment]
Good to merge!

**Comment by dnqb** [comment]
good to merge!

**Pull request merged**